ranger-zookeeper/src/main/java/io/appform/ranger/zookeeper/servicefinder/ZkNodeDataSource.java

```diff
@@ -93,8 +93,9 @@ private Optional<List<ServiceNode<T>>> checkForUpdateOnZookeeper(D deserializer)
         }
         catch (Exception e) {
             log.error("Error getting service data from zookeeper: ", e);
```

Comment:
Let's make this message also similar to what is the exception message.

Reply:
changed the error message, please review